tails_files/install.sh

```diff
@@ -1,17 +1,37 @@
 #!/bin/bash
+# SecureDrop persistent setup script for Tails 
 
 # check for root
 if [[ $EUID -ne 0 ]]; then
-    echo "This script must be run as root" 1>&2
-    exit 1
+	echo "This script must be run as root" 1>&2
```

Comment:
@ageis This is still a blocker for merge. Additionally, it appears the most recent commits have introduced more indentation inconsistencies below.